sensorprocess/sensorprocess_test.go

```diff
@@ -28,12 +29,21 @@ func TestStartOfflineSensorProcess(t *testing.T) {
 		ReadingTime: time.Now().UTC(),
 	}
 
+	odometerReading := s.TimedOdometerReadingResponse{
+		Position:    geo.NewPoint(5, 4),
```

Comment:
[q] arent these values defined in another file? If so they should be referenced from there

Reply:
Note: If youd prefer to have a separate def, id define these at the top of the file as they are used in multiple fucntions